darylldavidfst/fstbazaar
# Update User entity class

## Change Summary

- Updated User entity class:
   - removed firstName, lastName, and photo fields
   - Added auditing features (createdAt, updatedAt) for User
- Deleted Role entity class and related classes
- Updated view template:
  - removed First Name and Last Name columns in the Users list/table

## Test
1. Make sure fst_bazaar db is reset to have no tables
2. Run the application
3. Run `shouldSaveUser()` in `UserServiceTest`
